### DISCUSSION TITLE
SSPC website

### DISCUSSION THREAD

**Discussion by mrchristian** [singlesourcepub/community#12] [Q&A]
@tarleb can I ask you to put a Hugo theme on the website? https://singlesourcepub.github.io/community/

**Comment by tarleb** [comment]
To be honest, I don't really know how. @xldrkp has much more experience with Hugo.

**Comment by mrchristian** [comment]
Hi sure, sorry I was getting your users mixed up 😉

Simon Worthington - @mrchristian99 | Tag: Generation Research / Nacht: future publishing research | ORCID ID https://orcid.org/0000-0002-8579-9717
________________________________

**Reply by xldrkp** [reply]
@mrchristian Is Hugo still a thing to do?

**Comment by mrchristian** [comment]
I'd say its optional, see what you think. We only need a one page website for the moment to point to the other areas https://singlesourcepub.github.io/community/ | we will need Hugo eventually for translation, blogs, and if we have more pages. So depends on your time. Content is more important.

**Reply by xldrkp** [reply]
I spent my time on the content and some additions to the current site which is fine. Thanks for your work on that. Let's go with Hugo later. I agree with the translation thing.

**Reply by jcolomb** [reply]
Hello, 
The site would gain to have a definition of ssp.

if going to hugo, the learn theme might well be a good fit.

**Reply by mrchristian** [reply]
'learn' theme thanks. We did put up something very general - but it's not deatils as we acknowledge there are many different workflowsw and considerations https://singlesource.pub/ | Some are confined to multi-format typesetting, others will deal with the whole reproducible research publishing - like Daniel Nüst's recent paper https://zenodo.org/record/4818120 

A goal of the community will be to show off this different approaches - and in Autumn hold a Barcamp and publishing-Dash to document the different approaches.
